### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by Shivani-1524**
Added Valorant and Genshin quiz pages
Please review my javascript for the quiz pages.

### CODE REVIEW

**Review by eakansh125 — APPROVED** [review]
Could not find a syntactic issue, just some minor pixel to rem recommendations which can be ignored for now.
Design wise suggestion: Try to keep the elements centered, it gives a good look to the user. Here the question number, question and options can be aligned in a single path. Will give them a better look.

image

Good to be merged from my side.

File: quiz.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+.light-txt {
+  color: var(--creamy-color);
+}
+.white-txt {
+  color: var(--white);
+}
+
+.dark-txt {
+  color: black;
```
Comment (pradipgharge) [inline quiz.css]: use color variable

File: quiz.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.new-font {
+  font-family: "Audiowide", cursive;
+}
+
+.mb-20 {
+  margin-bottom: 20px;
```
Comment (pradipgharge) [inline quiz.css:35]: rems can be used here

**Review by pradipgharge — APPROVED** [review]
The code is neatly written, at some place you have used px for lengths and margins, you can change that to rems. Rest good to merge !

File: quiz.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+.question {
+  font-weight: 500;
+}
+
+.options li {
+  font-weight: 600;
+  height: max-content;
+  list-style-type: none;
+  padding: 0.7rem;
```
Comment (yaswanthmyneni) [inline quiz.css]: changes: use px for less than 1rem

**Review by rahulk31 — APPROVED** [review]
For the UI code seems to be well structured to me. Though I have few suggestions.
image

- To make the radio button and the label aligned on x-axis.
- Secondly, the background color for options can be made consistent using same color instead of different color for different options. If that is the theme of the game then I am not sure 😅 .
Just a suggestion, you can opt not to implement. Overall great work. Can be merged.

File: quiz.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+.btn-quiz-sbmt {
+  width: 40%;
+  height: 50px;
+  margin: auto;
+  max-width: 400px;
+}
+
+.content-container {
+  margin-bottom: 100px;
```
Comment (yaswanthmyneni) [inline quiz.css]: changes:  you can use the rem unit here.

**Review by yaswanthmyneni — APPROVED** [review]
UI looks great, you can merge it after changes.

**Comment by Shivani-1524** [comment]
> Could not find a syntactic issue, just some minor pixel to rem recommendations which can be ignored for now. Design wise suggestion: Try to keep the elements centered, it gives a good look to the user. Here the question number, question and options can be aligned in a single path. Will give them a better look.
> 
> image
> 
> Good to be merged from my side.

Okay will change px to rem and will consider changing the alignment accordingly. Thanks a lot 😁

File: quiz.css (hunk 3 above)
Comment (Shivani-1524) [reply quiz.css]: Done!

File: quiz.css (hunk 4 above)
Comment (Shivani-1524) [reply quiz.css]: Done!

**Comment by Shivani-1524** [comment]
> For the UI code seems to be well structured to me. Though I have few suggestions. image
> 
> * To make the radio button and the label aligned on x-axis.
> * Secondly, the background color for options can be made consistent using same color instead of different color for different options. If that is the theme of the game then I am not sure 😅 .
>   Just a suggestion, you can opt not to implement. Overall great work. Can be merged.

I thought using different colors will help with better differentiation between options a, b and c . I will try to implement the design suggestions in my next commits. Thanks a lot for the compliment!

File: quiz.css (hunk 2 above)
Comment (Shivani-1524) [reply quiz.css:35]: okay will do!

File: quiz.css (hunk 1 above)
Comment (Shivani-1524) [reply quiz.css]: okay will do!

**Pull request merged**